src/ParametricOptInterface.jl

```diff
@@ -425,11 +433,22 @@ function MOI.set(model::ParametricOptimizer, attr::MOI.ObjectiveSense, sense::MO
     return MOI.set(model.optimizer, attr, sense)
 end
 
+function MOI.get(model::ParametricOptimizer, attr::T, c::MOI.ConstraintIndex) where {
+                                                                                        T <: Union{
+                                                                                                MOI.ConstraintPrimal,
+                                                                                                MOI.ConstraintDual,
+                                                                                                }
+                                                                                    }
```

Comment:
bad indentation. see JuMP's style guide

Reply:
@joaquimg  Where in JuMP's documentation do they talk about indentation?
I went to https://jump.dev/JuMP.jl/stable/developers/style/#Style-guide and couldn't find much about it
Could you direct me to where I could find this information please?

Reply:
The answer to this is just to run JuliaFormatter over the code:
https://jump.dev/JuMP.jl/stable/developers/style/#JuliaFormatter